### DISCUSSION TITLE
Any ETA on General Availability (GA)?

### DISCUSSION THREAD

**Discussion by JuanMeeske** [Azure/data-api-builder#2156] [General]
The [roadmap ](https://github.com/Azure/data-api-builder/discussions/1377) document says that version 1 will be reached March 2024. 
Is there any update on the GA date for Data API Builder?

**Comment by Benjiiim** [comment]
"Beginning of may" was announced during the last community standup. Recording available at https://youtu.be/387Ay434vOI - Around 6:30.
